src/Analyzers/CSharp/CodeFixes/ConvertNamespace/ConvertNamespaceTransform.cs

```diff
@@ -156,11 +149,112 @@ public static (SourceText text, TextSpan semicolonSpan) ConvertNamespaceDeclarat
             return new TextChange(new TextSpan(textLine.Start, commonIndentation), newText: "");
         }
 
+        private static SourceText IndentNamespace(
+            ParsedDocument document, string indentation, SyntaxAnnotation annotation, CancellationToken cancellationToken)
```

Comment:
nit: i think it would be great if we extracted this as an indentation helper which takes a parsed doc, an indentatino, and a start/end line number, and does the work in there.

Reply:
➡️ I agree, and there's already a new bug filed that would benefit from it (#68953). However, we already have the outdent behavior contained in this file that would need to be extracted, so for now I'll keep the pair together and allow them to be refactored as a separate unit.